### PR TITLE
CHI1524 Case list styling

### PR DESCRIPTION
**Pull request by mythilytm**
Primary reviewer: @GPaoloni 

## Description
- This ticket required three corrections to the style. I was able to work on the following 2 issues:
    - Checkboxes Checkboxes
    - Case list header cells to be flush with the lines around -Screenshot 2022-11-22 at 11 12 57 PM
    - Table Container width to fit 1280px
    
### Checklist
- [ ] Corresponding issue has been opened
- [ ] New tests added
- [ ] Feature flags added
- [ ] Strings are localized
- [ ] Tested for chat contacts
- [ ] Tested for call contacts

### Related Issues
Fixes # 1524

### CODE REVIEW

**Comment by acedeywin** [comment]
@GPaoloni @mythilytm I have completed this task.

File: plugin-hrm-form/src/styles/caseList/index.ts [hunk 1]
```diff
@@ -52,18 +52,32 @@ export const CLTableRow = withStyles({
...
     overflow: 'hidden',
-    paddingLeft: '10px',
+    padding: '0 0 0 10px',
   },
   body: {
     verticalAlign: 'top',
     paddingTop: 8,
   },
 }))(TableCell);
+CLHeaderTableCell.displayName = 'CLTableCell';
```
Comment (stephenhand) [inline plugin-hrm-form/src/styles/caseList/index.ts:67]: Should this not be 'CLHeaderTableCell'?

**Comment by mythilytm** [comment]
@acedeywin Regarding the width, the requirement is to have the table fit to 1280px

**Comment by mythilytm** [comment]
> @acedeywin Regarding the width, the requirement is to have the table fit to 1280px

The change you made helped though. I tried every other thing and thought I tried width :P

**Review by stephenhand — APPROVED** [review]
LGTM

**Pull request merged**